### ISSUE TITLE
Parse error on gcovr xml file

### ISSUE THREAD

**Issue by sarnold** [irongut/CodeCoverageSummary#9]
## Bug Report

Using [gcovr cobertura format](https://gcovr.com/en/5.0/guide.html#cobertura-xml-output):

Parse Error: Input string was not in a correct format.

### Workflow File

Before action step:

```
gcovr -r src/ --xml-pretty -o coverage.xml
```

```yaml
    - name: Code Coverage Summary Report (data)
      uses: irongut/CodeCoverageSummary@v1.0.3
      with:
        filename: coverage.xml
        output: 'both'
```

### Actions Log
```
Run irongut/CodeCoverageSummary@v1.0.3
/usr/bin/docker run --name ghcrioirongutcodecoveragesummaryv102_3f9672 --label e1cc51 --workdir /github/workspace --rm -e PYTHONIOENCODING -e PIP_DOWNLOAD_CACHE -e pythonLocation -e LD_LIBRARY_PATH -e INPUT_FILENAME -e INPUT_OUTPUT -e INPUT_BADGE -e INPUT_FORMAT -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/redis-ipc/redis-ipc":"/github/workspace" ghcr.io/irongut/codecoveragesummary:v1.0.2  "coverage.xml" "--badge" "false" "--format" "text" "--output" "both"
Code Coverage File: coverage.xml
Parse Error: Input string was not in a correct format.
Error: Parsing code coverage file.
```

### Expected Behavior
A coverage report for the following steps. ``<dry laughter>``

### Additional Context
Full workflow logs and yaml are all [here](https://github.com/VCTLabs/redis-ipc/actions/runs/1272311030) with coverage data artifact.

I just updated to v1.0.3 while debugging, although the container still says v1.0.2 (maybe your new package isn't fully deployed yet?). I tried fixing the (missing) package name in the coverage data but that didn't help, and I don't have much error detail in my logs.  Thanks!

**Comment by irongut** [comment]
@sarnold Thanks for including the link to your workflow run.

The problem is the `package` line in the test report:

```xml
<package name="" line-rate="0.6031468531468531" branch-rate="0.3968253968253968" complexity="0.0">
```

There are two issues with this line - the missing `name` is a minor issue but the problem is `complexity` which I expect to be an integer, not a floating point value. I can't find a spec for the Cobertura format and the [DTD](https://raw.githubusercontent.com/cobertura/web/master/htdocs/xml/coverage-04.dtd) doesn't specify data types so the action will need to handle both.

**Comment by sarnold** [comment]
I also ran into the cosmetic variant of package name="." with a python package and coverage.py (which is annoying, but easy enough to fix).  Your action parser is happy with name="." and puts it in the report, but coverage.py should also be able to find the name of a py_module without a package dir; I need to go file a bug with coverage.py for that...

**Comment by sarnold** [comment]
With a temp sed :knife: to force "0" my workflow completes nicely. Thanks! :rocket:

**Comment by irongut** [comment]
@sarnold I've released v1.0.5 which should work for you.
